From c294ebd17bcc3138edc87523195c877c3b094205 Mon Sep 17 00:00:00 2001
From: Dmitry Artamonow <mad_soft@inbox.ru>
Date: Fri, 13 Mar 2009 22:30:15 +0300
Subject: [PATCH 26/28] h3600: add micro platform device

So 'micro' MFD driver can be used for backlight, touchscreen, buttons
and battery.

Signed-off-by: Dmitry Artamonow <mad_soft@inbox.ru>
---
 arch/arm/mach-sa1100/h3600.c |    8 +++++++-
 1 files changed, 7 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-sa1100/h3600.c b/arch/arm/mach-sa1100/h3600.c
index c7e0458..ee85530 100644
--- a/arch/arm/mach-sa1100/h3600.c
+++ b/arch/arm/mach-sa1100/h3600.c
@@ -449,8 +449,14 @@ static struct platform_device h3600_keys = {
 	},
 };
 
+struct platform_device h3600_micro_device = {
+	.name	= "h3600-micro",
+	.id	= -1,
+};
+
 static struct platform_device *h3600_devices[] = {
-       &h3600_keys,
+	&h3600_keys,
+	&h3600_micro_device,
 };
 
 static void h3600_mach_init(void)
-- 
1.6.1.3

